src/app/banquet/page.js

```diff
@@ -1,12 +1,39 @@
 import React from "react";
 import Title from "@/components/Title";
 import Description from "@/components/Description";
+import Image from "next/image";
 
 const page = () => {
   return (
     <div className="w-full overflow-clip flex flex-col justify-center items-center gap-10">
-      <Title />
-      <Description />
+      <Title text="Banquet" />
+      <Description
+        title={"What is Banquet?"}
+        text={
+          "Banquet is our end of the year celebration! It is a time for all of us to gather together and look back at the wonderful year we have had together."
+        }
+      />
+      <div className="flex justify-center items-center w-full h-60">
+        <Image
+          src="/images/image4.webp"
```

Comment:
you should import the image at the top, and use that here. Check other files for examples